src/fn/upload-custom-translations.js

```diff
@@ -24,11 +23,11 @@ module.exports = (projectDir, couchUrl) => {
         .filter(name => FILE_MATCHER.test(name))
         .map(fileName => {
           const id = idFor(fileName);
-          const language_code = id.substring(id.indexOf('-') + 1);
-          const language_name = ISO639.getName(language_code);
+          const language_code = id.substring('messages-'.length);
+          let language_name = ISO639.getName(language_code);
```

Comment:
Should we use the `getNativeName` interface? Or even the same format used by the webapp?  `Francais (French)` or `${ISO639.getNativeName} (${ISO639.getName})`

Reply:
Won't it be redundant to have both a native and english language name? Both represent the same information

Reply:
I vote for consistency